src/pages/groupComparison/comparisonGroupManager/GroupCheckbox.tsx

```diff
@@ -169,73 +112,11 @@ export default class GroupCheckbox extends React.Component<
                         )}
                         onClick={this.onCheckboxClick}
                     />
-                    {this.editingName ? this.editingNameUI : this.label}
+                    {this.label}
                 </div>
             );
         }
 
-        let editingRelatedIcons;
```

Comment:
we can no longer rename groups?

Reply:
yes, this is because we are allowing sharing of groups